### ISSUE TITLE
Trial run populated IRS 1095-B from widget with screen reader

### ISSUE THREAD

**Issue by tameracorson** [department-of-veterans-affairs/va.gov-team#42974]

**Comment by tameracorson** [comment]
![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/d8377122-5e43-4f09-a15c-da3326ecd9cc)

**Comment by tameracorson** [comment]
Test Blank PDF
[https://app.zenhub.com/files/133843125/cdfcf553-1499-4075-888a-c9d413ae2d97/download](https://app.zenhub.com/files/133843125/cdfcf553-1499-4075-888a-c9d413ae2d97/download)

**Comment by tameracorson** [comment]
Test #1 
[https://app.zenhub.com/files/133843125/f42c8dd6-c915-4372-9ff9-9cb6e6a983fc/download](https://app.zenhub.com/files/133843125/f42c8dd6-c915-4372-9ff9-9cb6e6a983fc/download)

**Comment by tameracorson** [comment]
Test #2 
[https://app.zenhub.com/files/133843125/c2889527-ed68-40dc-80d9-8cb25b8e328c/download](https://app.zenhub.com/files/133843125/c2889527-ed68-40dc-80d9-8cb25b8e328c/download)

**Comment by tameracorson** [comment]
Flattened data failed during online test with screen reader 6/22/2022

**Comment by tameracorson** [comment]
Unflatten data test - Failed 6/23/2022
 ![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/9896cc43-d9de-46c7-9d64-bad8d56f71aa)

**Comment by tameracorson** [comment]
PDF output from widget without privacy data - partial fail
![image.png](https://images.zenhubusercontent.com/61856bab21cd5965714d703d/0397a633-9d6a-4f3a-9c33-7dd8307b8d60)
